handlers/middlewares.go

```diff
@@ -56,7 +65,26 @@ func (h *Handler) AuthMiddleware(next http.Handler) http.Handler {
 			// }
 			// return
 			if provider.GetProviderType() == models.RemoteProviderType {
-				provider.Logout(w, req)
+				if err.Error() == "http: named cookie not present" {
+					provider.Logout(w, req)
```

Comment:
Earlier we had the problem for wildcard response set, 
![](https://user-images.githubusercontent.com/79860504/150266531-36405c71-471f-43ff-b4f5-e587d842d960.png)
Are we solving it now?

Reply:
Yes, this is solved with the proposed changes, as we will not redirect from the server in case of an invalid session, instead query will fail, and UI will handle it by alerting the user.